lib/ProductOpener/Web.pm

```diff
@@ -291,14 +298,21 @@ The data structured can be passed to HTML templates to construction a list of op
 
 sub get_languages_options_list ($target_lc) {
 
+	return $lang_options_cache{$target_lc} if (defined $lang_options_cache{$target_lc});
+
 	my @lang_options = ();
 
 	my %lang_labels = ();
 	foreach my $l (@Langs) {
```

Comment:
It wasn't changed in this PR, but can we rename $l to $language_code or something? In the GitHub font I can't really tell $l from $1 : `$l / $1`